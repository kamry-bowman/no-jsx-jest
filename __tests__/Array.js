const method = require('../Array');

test('snapshot', () => {
    expect(method()).toMatchSnapshot();
});