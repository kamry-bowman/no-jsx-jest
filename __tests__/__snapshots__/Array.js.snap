// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Array [
  1,
  2,
  3,
  4,
  5,
]
`;
